cloudpickle_generators/tests/test_segfault.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import cloudpickle
+import cloudpickle_generators
+cloudpickle_generators.register()
```

Comment:
This happens in `conftest.py`, so it should already be executed before the tests run.